sdk/evm/src/ntt.ts

```diff
@@ -117,14 +159,62 @@ export class EvmNtt<N extends Network, C extends EvmChains>
       this.provider
     );
 
-    this.xcvrs = [
-      // Enable more Transceivers here
-      new EvmNttWormholeTranceiver(
-        this,
-        contracts.ntt.transceiver.wormhole!,
-        abiBindings!
-      ),
-    ];
+    if (contracts.ntt.transceiver.wormhole != null) {
+      this.xcvrs = [
+        // Enable more Transceivers here
+        new EvmNttWormholeTranceiver(
+          this,
+          contracts.ntt.transceiver.wormhole,
+          abiBindings!
+        ),
+      ];
+    } else {
+      this.xcvrs = [];
+    }
+  }
+
+  async getTransceiver(ix: number): Promise<NttTransceiver<N, C, any> | null> {
+    // TODO: should we make an RPC call here, or just trust that the xcvrs are set up correctly?
+    return this.xcvrs[ix] || null;
+  }
+
+  async getMode(): Promise<Ntt.Mode> {
+    const mode: bigint = await this.manager.getMode();
+    return mode === 0n ? "locking" : "burning";
+  }
+
+  async isPaused(): Promise<boolean> {
+    return await this.manager.isPaused();
+  }
+
+  async *pause() {
+    const tx = await this.manager.pause.populateTransaction()
+    yield this.createUnsignedTx(tx, "Ntt.pause");
+  }
+
+  async *unpause() {
+    const tx = await this.manager.unpause.populateTransaction()
+    yield this.createUnsignedTx(tx, "Ntt.unpause");
+  }
+
+  async getOwner(): Promise<AccountAddress<C>> {
+    return new EvmAddress(await this.manager.owner()) as AccountAddress<C>;
+  }
+
+  async *setOwner(owner: AccountAddress<C>) {
+    const canonicalOwner = canonicalAddress({chain: this.chain, address: owner});
```

Comment:
The parameter here could be `AnyEvmAddress` to allow for string to be passed as well

and the `canonicalOwner` could be set as `new EvmAddress(owner).toString()`

Reply:
oo, that's neat. wasn't aware of that type